showkase-processor/src/main/java/com/airbnb/android/showkase/processor/ShowkaseProcessor.kt

```diff
@@ -112,7 +129,14 @@ class ShowkaseProcessor @JvmOverloads constructor(
         val skipPrivatePreviews = environment.options["skipPrivatePreviews"] == "true"
         return roundEnvironment.getElementsAnnotatedWith(PREVIEW_CLASS_NAME)
             .mapNotNull { element ->
-                if (showkaseValidator.checkElementIsAnnotationClass(element)) return@mapNotNull null
+                if (showkaseValidator.checkElementIsAnnotationClass(element)) {
```

Comment:
Since we are explicitly letting Showkase know which custom annotations are meant for preview, I think we can return null and skip them for processing, correct? 🤔

If this was working, then we wouldn't need the use annotation options at all. iirc, this was only working for ksp and not kapt, correct?

Reply:
I think when I tested the KAPT version of this without the compiler options, I tested it with repeatable annotations which do not work with KAPT anyways.  This was totally my mistake and might have lead to confusion.  I think we can ditch the compiler argument strategy and use this instead. IMO, this will be a better user experience for the enduser. Wdyt?